Formula/mk-configure.rb

```diff
@@ -0,0 +1,22 @@
+class MkConfigure < Formula
+  desc "Lightweight replacement for GNU autotools"
+  homepage "https://github.com/cheusov/mk-configure"
+  url "https://downloads.sourceforge.net/project/mk-configure/mk-configure/mk-configure-0.29.3/mk-configure-0.29.3.tar.gz"
+  sha256 "1f483f717a09d06c82a6f3f284dd34234f54ead032e6987d33f3cf8109490c05"
+
+  depends_on "bmake"
+  depends_on "makedepend"
+
+  def install
+    ENV["PREFIX"] = prefix.to_s
+    ENV["MANDIR"] = man.to_s
+
+    system "bmake", "all"
+    system "bmake", "install"
+    (share/"doc/mk-configure").install("presentation/presentation.pdf")
+  end
+
+  test do
+    system "#{bin}/mkcmake", "-V", "MAKE_VERSION", "-f", "/dev/null"
```

Comment:
This needs to be a test that exercises and verifies the functionality of the software.

Reply:
I implemented "test" target but I don't know how to run it for test. Can you help me please?

Reply:
For now test target looks like this

  test do
    ENV["PREFIX"] = prefix.to_s
    ENV["MANDIR"] = man.to_s
    ENV["NOSUBDIR"] = "lua_dirs hello_lua hello_lua2 hello_lua3"

    system "bmake", "all"
    system "bmake", "test"
  end

Reply:
Also, I'd like to say that "mkcmake -V MAKE_VERSION" tests mk-configure to some extent.